### PR TITLE
Improve core-html-elements style

### PR DESCRIPTION
**Pull request by sourabhv**
Move info about `style` attr below style element. It was kinda counter-intuitive to start with element and give information about style attr first

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - web/html/core-html-elements/style-element.md

File: web/html/core-html-elements/style-element.md [hunk 1]
```diff
@@ -20,65 +20,78 @@ links:
...
 ---
+
 # Style element
 
 ---
+
 ## Content
 
-The HTML `style` element is used to style the content of an html document. Also, there is a `style` attribute used to achieve the same thing, but per element.
+The HTML `style` element is used to style the content of an html document. These styles are loaded with the HTML page and work even if external CSS fails to load for some reasno. These are often used to load what we call **Critical CSS**, without which page is really hard to read
```
Comment (kapnobatai137) [inline web/html/core-html-elements/style-element.md]:
```suggestion
The HTML `style` element is used to style the content of an html document. These styles are loaded with the HTML page and work even if external CSS fails to load for some reason. These are often used to load what we call **Critical CSS**, without which the page is really hard to read
```

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - web/html/core-html-elements/style-element.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - web/html/core-html-elements/style-element.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/core-html-elements/style-element.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/core-html-elements/style-element.md?ref=59b463c0be7bd135f0f76f7f724c4db02cb2cd5b)

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - web/html/core-html-elements/style-element.md

**Pull request merged**